README.md

```diff
@@ -258,43 +258,23 @@ The examples can be found in the [examples folder](https://github.com/getyoti/ph
 ### Profile sharing
 
```

Comment:
Do you think it would be useful to have a section something like this here so that the users will know that they will need Docker before they get to the penultimate step:


#### Requirements
* [Docker](https://www.docker.com/community-edition)

Reply:
I don't think we need to specify that, but will just add in the title.

Reply:
Would be better if we linked to the shortest possible instructions for installing Docker, but maybe later